cpp/jvm-host/src/host_jni.cpp

```diff
@@ -434,6 +434,13 @@ JNIEXPORT jint JNICALL Java_com_r3_conclave_host_internal_Native_getQuoteDCAP(JN
     return (int)eval_result;
 }
 
+jbyteArray jbyteArrayFromCharPtr(JNIEnv *jniEnv, unsigned size, char* data) {
+    unsigned sizeActual = size - 1; // Drop the \0 terminator
```

Comment:
Should we make it clear whether these are `unsigned long` or `unsigned int`? Are all this `*size` `uint32`? I don't think that a potential casting is a problem, just double checking.
Also, should we put defensive code in case for whatever reason this is called with size = 0?

Reply:
Added some checks